WTWSMS/events/WtWSMS_schisms.txt

```diff
@@ -271,6 +271,22 @@ character_event = {
 		set_parent_religion = { religion = ebionites parent = orthodox }
 		set_parent_religion = { religion = marcionism parent = orthodox }
 		set_parent_religion = { religion = binitarian parent = orthodox }
+		set_parent_religion = { religion = melchisidechianism parent = orthodox }
+		set_parent_religion = { religion = sethian parent = catholic }
+		set_parent_religion = { religion = eutychian parent = nestorian } #Trinitarian beliefs extremely similar to Nestorianism
+		set_parent_religion = { religion = antinomianism parent = nestorian } #Antinomianism is one of the most ancient sects, and most like the Messalians that are also Antinomians, and Messalians are counted as Nestorian heresies
+		set_parent_religion = { religion = monatist parent = catholic }
+		set_parent_religion = { religion = total_depravity parent = catholic }
+		set_parent_religion = { religion = apollinarian parent = orthodox } #Similar to Monothelites ("Jesus is controlled by Skynet", AKA no mind of His own)
+		set_parent_religion = { religion = adoptionism parent = miaphysite } #Monarchism is a Miaphysite heresy already, this is a branch of it built on Jesus' baptism
+		set_parent_religion = { religion = docetic parent = catholic }
+		set_parent_religion = { religion = pneumatomachians parent = orthodox } #Founded by Macedonius, a Bishop of Constantinople
+		set_parent_religion = { religion = patripassianism parent = miaphysite } #Monarchism is a Miaphysite heresy already, and this is a form of it
+		set_parent_religion = { religion = sabellianism parent = miaphysite } #Moanrchists again
+		set_parent_religion = { religion = psilanthropism parent = orthodox }
+		set_parent_religion = { religion = priscillianism parent = zoroastrian } #This is basically Maneachism, and in fact, it should be a heresy of that already
+		
+		
```

Comment:
Moving around many heresies in an event that will be redone, without factoring in the schism in question, won't work. Furthermore, your motivation being "should already be in this group" does not work